crawl-ref/source/dat/des/branches/swamp.des

```diff
@@ -1844,6 +1844,31 @@ _.tt.tttt.t.tt.ttttt_
             -_-
 ENDMAP
 
+#### A dangerous temple to kiku
+NAME: beargit_kiku_chapel
+DEPTH: Swamp:2-
+TAGS: no_monster_gen
+KMONS: 1 = bunyip zombie w:10 / bunyip skeleton w:10 / bog body w:20 / death scarab w:1
+KMONS: 2 = patrolling juggernaut zombie w:20 / patrolling caustic shrike zombie w:3 / patrolling shard shrike zombie w:3 / patrolling spectral juggernaut w:5 / patrolling spectral caustic shrike w:1
```

Comment:
These zombies are a bit too strong for swamp. I get that you want to have a big bad entry guardian when the door comes down. Consider a mix of spectral spriggans, weighted towards a berserker for the flavour.
```suggestion
MONS: patrolling spectral spriggan berserker w:30 / patrolling spectral spriggan defender / patrolling spectral spriggan air mage
```

Reply:
Can we keep a low chance of juggernaut zombie because those are allowed on D:15 (depths entrance vault)?

Reply:
See latest commit for what I think is balanced. Spectral spriggans are complete pushovers so I made them simulacrum. I had a chat with gammafunk and he doesn't think rare Zuggernauts or caustic shrike zombies are out of place in swamp, but I made them rare. Also randomized the blood fountain and necromancy spell levels.